### ISSUE TITLE
Export Error

### ISSUE THREAD

**Issue by Beppo38** [sqlitebrowser/sqlitebrowser#2246]
Details for the issue
--------------------

#### What did you do?
Export of 437 Entries as JSON

#### What did you expect to see?
No Error 

#### What did you see instead?
But the JSON only contains 90 Entries 

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [ ] Windows: ( _version:_ ___ )
- [ ] Linux: ( _distro:_ ___ )
- [X ] macOS: ( _version:10.15.4)
- [ ] Other: ___

#### What is your DB4S version?

- [ ] 3.12.0-(alpha or betas)
- [ ] 3.11.2
- [ X] 3.11.1
- [ ] 3.10.1
- [ ] Other: ___

#### Did you also

- [ ] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [ ] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by horst-p-w-neubauer** [comment]
@Beppo38 , I've just successfully exported a table with 23.584 Entries. Maybe there's something special with a value of you're 91th entry. If you need help to find it, you might upload your db perhaps just a test-version stripped down to the one table - to https://dbhub.io/

**Comment by justinclift** [comment]
Ahhh.  This could be another case of the original JSON exporter library we used (that was in version 3.11.x) being broken.

We've since replaced the JSON export with a different, much more robust JSON exporting library.  That's present in our recent 3.12.0-alpha1 release, and in our nightly builds.

If you're ok to try either of those instead, either (in theory) should work ok. :grin: 

* https://sqlitebrowser.org/blog/first-alpha-release-for-3-12-0/
* https://nightlies.sqlitebrowser.org/latest/DB.Browser.for.SQLite-sqlcipher.dmg
